### ISSUE TITLE
Widgets

### ISSUE THREAD

**Issue by LucasGGamerM** [LucasGGamerM/moshidon#228]
Allow for viewing timelines in widgets

**Comment by dtantono** [comment]
Yes, it will be wonderful when we go to our home screen. we can see our homepage and receive periodic update easily without visit the app directly.

**Comment by inson1** [comment]
What about widget for how many followers I have
or how many favorites I got

plus maybe in how many in set time - last hour, day, week, month, year

**Comment by inson1** [comment]
and something for controlling my instance
- how many accounts I have on it
- how many favorites posts on instance got
- how many posts I got on my instance

plus how many in last hour, day, week, month, year
- new accounts 
- new favorites
- new posts
